lib/pages/chat/events/message.dart

```diff
@@ -188,6 +188,7 @@ class Message extends StatelessWidget {
                                                           .labelMedium
                                                           ?.copyWith(
                                                             fontWeight: FontWeight.w500,
+                                                            fontFamily: 'Roboto',
```

Comment:
IMO: pls make it flexible, if it is not flexible I think it will be very hard to manage